homeassistant/components/camera/rpi_camera.py

```diff
@@ -36,7 +37,7 @@
 DEFAULT_VERTICAL_FLIP = 0
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Optional(CONF_FILE_PATH): cv.string,
+    vol.Optional(CONF_FILE_PATH, default=DEFAULT_FILE_PATH): cv.string,
```

Comment:
I guess that with [`cv.isfile`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/config_validation.py#L99) instead of `cv.string` we could get rid of the checks in `setup_platform()`.

Reply:
Yep, seems like it! Will change in a few :smile: